index.js

```diff
@@ -2,14 +2,10 @@ const Configuration = require('./lib/configuration')
 const Context = require('./lib/context')
 
 module.exports = robot => {
-  robot.on('*', receive)
-
-  async function receive (event) {
-    if (event.payload.repository) {
-      const github = await robot.auth(event.payload.installation.id)
-      const context = new Context(github, event)
-      const config = await Configuration.load(context, '.probot.js')
-      return config.execute()
-    }
-  }
+  robot.on('*', async context => {
+    const github = await robot.auth(context.event.payload.installation.id)
+    const contextOld = new Context(github, context.event)
```

Comment:
I don't think `context.event` is right here since that's only the event name and not the payload.

Reply:
Good catch! Dumb mistake. Fixed!